migrations/sql/V1.7__add_document_status.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE mine_expected_document_status
```

Comment:
Hi there! I was just taking a look at the data model and noticed a couple of minor things about this lookup table. Sorry I didn't catch it earlier, so if there are large risks or time constraints with an of my suggestions, please let me know. These are not "must do's" but they are things that will help to keep consistency within the schema.

1) Can you please ensure the table has a TABLE COMMENT.

2) Table naming... to help make this lookup table consistent with others in the MDS schema, can I suggest that the table be named **mine_expected_document_status_type**... adding the "type" at the end will help to provide a better understanding that this is a lookup table, rather than the status of the expected_document. For all lookup tables, I'd like the naming to be suffixed with either code or type, depending on whether or not the lookup values are coded values or "real words" values.

3) Another item re: consistency, the code tables in MDS have a column that holds the lookup value and then an additional description column... this table only has the description column... having 2 columns in this table may not be necessary if there is no need to provide a code and a description for the status... I think other status code tables in MDS have used coded values though (i.e. mine_operation_status_code is a good example). That all said, if coded values aren't required and you are planning to use real words like "Active", then I'd suggest that we change the name of the "description" column to something like "expected_document_status_type" or "expected_document_status_name".

Happy to chat any time! Thanks.